dsnp/graphChange.ts

```diff
@@ -0,0 +1,36 @@
+export default {
+  type: "record",
+  name: "GraphChange",
+  fields: [
+    {
+      // Do we need this?
+      name: "announcementType",
```

Comment:
Unlike the parquet one, I feel like we don't need announcementType for the on-chain data. Thoughts?

Reply:
The only reason I could think of to keep it would be for completeness, to reference back to the DSNP spec.  But then we can always document which are on chain.